spec/models/user_spec.rb

```diff
@@ -57,6 +57,20 @@
     end
   end
 
+  describe "Scopes >" do
+    describe "->alphabetized" do
+      let!(:a_name_user) { create(:user, name: 'Amanda') }
+      let!(:z_name_user) { create(:user, name: 'Zachary') }
+
+      it "retrieves only those with active status" do
+        alphabetized_list = described_class.alphabetized
```

Comment:
there were a couple of other ways I could have accomplished this that would look a little more readable; I chose to go a more performant route